src/utils.jl

```diff
@@ -24,9 +24,16 @@ function facedofs(cell, local_face_idx::Int, field_idx::Int)
 end
 
 Ferrite.vertices(cell::Ferrite.Cell{3,3,1}) = cell.nodes
+
 Ferrite.default_interpolation(::Type{Ferrite.Cell{3,3,1}}) = Ferrite.Lagrange{2,Ferrite.RefTetrahedron,1}()
-face_cell(cell::Ferrite.Tetrahedron, i::Int) =  Ferrite.Cell{3,3,1}(Ferrite.faces(cell)[i])
-face_cell(cell::Ferrite.Hexahedron, i::Int) = Ferrite.Quadrilateral3D(Ferrite.faces(cell)[i])
+
+# Note: This extracts the face spanned by the vertices, not the actual face!
+linear_face_cell(cell::Ferrite.Tetrahedron, local_face_idx::Int) =  Ferrite.Cell{3,3,1}(Ferrite.faces(cell)[local_face_idx])
+linear_face_cell(cell::Ferrite.Hexahedron, local_face_idx::Int) = Ferrite.Quadrilateral3D(Ferrite.faces(cell)[local_face_idx])
```

Comment:
```suggestion
linear_face_cell(cell::Cell{3,N,6}, local_face_idx::Int) where N = Ferrite.Quadrilateral3D(Ferrite.faces(cell)[local_face_idx])
```

Is there a reason why we restrict this to these two cells? Should hold true as a hotfix for any order `Hexahedron` shouldn't it?

Reply:
Obviously. I derped here. :laughing: The restriction stems from the fact that some elements in >2D have different kinds of faces (e.g. pyramidal elements).

Reply:
I know. I will push the fixes next week, just had not found the time yet.